easybuild/easyblocks/t/tensorflow.py

```diff
@@ -69,6 +69,7 @@ def extra_options():
         extra_vars = {
             # see https://developer.nvidia.com/cuda-gpus
             'cuda_compute_capabilities': [[], "List of CUDA compute capabilities to build with", CUSTOM],
+            'package_filter': [[], "List of paths to filter out", CUSTOM],
```

Comment:
It's actually a list of filter *patterns* for paths  (since you're checking with `if fil not in p`), please clarify that and maybe also rename to `cpath_libpath_filter_patterns` to make it more explicit?